lib/checkPullRequestJob.js

```diff
@@ -32,6 +32,163 @@ const JOB_REGEX = new RegExp(
+  });
+
+  return registryFile && registryFile.patch.includes(job);
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} newJobFiles
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} changedFiles
+ */
+const getCronRegistryReminder = (newJobFiles, changedFiles) => {
+  let registryReminder = ', ';
+
+  const hasNotAddedFileToRegistry = newJobFiles.some(
+    (file) => {
+      const jobs = getNewJobsFromFile(file);
+      const jobModuleName = file.filename;
+      return jobs.some((job) => !isCronRegistryUpdated(
+        `${jobModuleName}.${job}`, changedFiles));
+    }
+  );
+
+  if (hasNotAddedFileToRegistry) {
+    const jobText = newJobFiles.length > 1 ? 'jobs' : 'job';
+    const jobRegistryFile = 'job registry'.link(
+      'https://github.com/oppia/oppia/blob/develop/cron.yaml');
+    registryReminder +=
+      'please add the new ' + jobText + ' to the ' + jobRegistryFile + ' and ';
+  }
+
+  return registryReminder;
+};
+
+/**
+ * @param {import('probot').Octokit.PullsListFilesResponseItem[]} newJobFiles
+ */
+const getCronJobNameString = (newJobFiles) => {
+  return getNameString(
+    newJobFiles,
+    {
+      singular: 'job',
+      plural: 'jobs'
+    },
+    CRON_JOB_REGEX
+  );
+};
+
+
+const getCronCommentBody = (newJobFiles, changedFiles, prAuthor) => {
+  const registryReminder = getCronRegistryReminder(newJobFiles, changedFiles);
+  const jobNameString = getCronJobNameString(newJobFiles);
+  const newLineFeed = '<br>';
+  const cronJobsForm = 'cron jobs form'.link(
+    '');// some link
+  const wikiLinkText = (
+    'this guide'.link(
+      'https://github.com/oppia/oppia/wiki/Running-jobs-in-production' +
+      '#submitting-a-pr-with-a-new-job'));
+
+  // This function will never be called when there are no job files.
+  if (newJobFiles.length === 1) {
+    const serverAdminPing = (
+      'Hi @' + serverJobAdmins.join(', @') +
+      ', PTAL at this PR, ' + 'it adds a new cron job.' + jobNameString
+    );
+
+    const prAuthorPing = 'Also @' + prAuthor + registryReminder +
+      'please make sure to fill in the ' + cronJobsForm +
+      ' for the new job to be tested on the backup server. ' +
+      'This PR can be merged only after the test run is successful. ';
+
+    return (serverAdminPing + newLineFeed + prAuthorPing +
+      newLineFeed + 'Thanks!');
+  } else {
+    const serverAdminPing = (
+      'Hi @' + serverJobAdmins.join(', @') +
+      ', PTAL at this PR, ' + 'it adds new cron job.' + jobNameString);
+
+    const prAuthorPing = 'Also @' + prAuthor + registryReminder +
+      'please make sure to fill in the ' + cronJobsForm +
+      ' for the new jobs to be tested on the backup server. ' +
+      'This PR can be merged only after the test run is successful. ' +
+      'Please refer to ' + wikiLinkText + ' for details.';
+
+    return (serverAdminPing + newLineFeed + prAuthorPing +
+      newLineFeed + 'Thanks!');
+  }
```

Comment:
Why is this if/else needed? The comments seem the same.

Reply:
Removed it!